spec/schemas/_common.aggregations.yaml

```diff
@@ -954,7 +954,7 @@ components:
             to_as_string:
               type: string
             key:
-              description: The bucket key. Present if the aggregation is _not_ keyed
+              description: The bucket key. Present if the aggregation is not keyed.
```

Comment:
This does lose the emphasis on "not", is there someway to get Vale to accept markdownisms or is there an explicit reason to deny this?

Reply:
This is the only place we do this AFAIK, I decided to remove it.